src/components/Flag/Flag.tsx

```diff
@@ -27,11 +28,24 @@ const getSourceByIsoCode = (isoCode: MuiTelInputCountry | null) => {
   return ''
 }
 
-const Flag = ({ size = 'small', isoCode, countryName = '' }: FlagProps) => {
+const defaultGetFlagSources: GetFlagSources = (isoCode, width) => {
+  return {
+    png: `https://flagcdn.com/w${width}/${isoCode}.png`,
+    webp: `https://flagcdn.com/w${width}/${isoCode}.webp`
+  }
+}
+
+const Flag = ({
+  size = 'small',
+  isoCode,
+  countryName = '',
+  getFlagSources = defaultGetFlagSources
+}: FlagProps) => {
   const isoCodeFormatted = isoCode ? isoCode.toLowerCase() : ''
   const sourceFound = getSourceByIsoCode(isoCode)
   // see https://flagpedia.net/download/api for the valid width
   const width = size === 'small' ? 40 : 80
+  const { webp, png } = getFlagSources(isoCodeFormatted, width)
```

Comment:
Nah, just map on keys of the object, you could have more format (jpg, jpeg, etc...)

Reply:
I can do that, but then we need to extend the return of `getFlagSources` with the source type right?

Reply:
LGTM